### DISCUSSION TITLE
Private websites Option

### DISCUSSION THREAD

**Discussion by Bashamega** [vercel/community#2913] [Idea]
Hello：）
So lately I have been using vercel, then today I have noticed that if I share the link for a private repo it opens.
So my idea is that make this feature in the settings of deployment.
like a check box.

**Comment by smaeda-ks** [comment]
@Bashamega Could you elaborate on that more? What do you mean by "if I share the link for a private repo it opens"?

If you want to make your preview links private, you can enable Deployment Protection:
https://vercel.com/docs/concepts/deployments/deployment-protection
